packages/report-server/src/__tests__/reportBuilder/transform/aliases.test.ts

```diff
@@ -88,4 +89,14 @@ describe('aliases', () => {
     });
     expect(transform(SINGLE_ANALYTIC)).toEqual([{ ...SINGLE_ANALYTIC[0], numberOfFacilities: 14 }]);
   });
-});
+
+  it('insertSummaryRowAndColumn', () => {
+    const transform = buildTransform(['insertSummaryRowAndColumn']);
+    expect(transform(MULTIPLE_ANALYTICS_SUMMARY_BINARY)).toEqual([
+      { summaryColumn: '75.0%', dataElement: 'Male condoms', TO: 'N', FJ: 'N', NR: 'Y', KI: 'N' },
+      { summaryColumn: '25.0%', dataElement: 'Female condoms', TO: 'N', FJ: 'Y', NR: 'Y', KI: 'Y' },
+      { summaryColumn: '0.0%', dataElement: 'Injectable contraceptives', TO: 'Y', FJ: 'Y', NR: undefined, KI: undefined },
+      { TO: '66.7%', FJ: '33.3%', NR: '0.0%', KI: '50.0%' },
```

Comment:
I may have gotten a little confused somewhere but I thought the numerator is meant to be `# of Ys / # of Ys and # of Ns`?

Reply:
@rohan-bes it's actually the opposite. It's the percentage of Ns, because they want to display how much isn't in stock.

Reply:
Ah! I see, all good then 👍